google-cloud-spanner/lib/google/cloud/spanner/admin/database/v1/doc/google/iam/v1/policy.rb

```diff
@@ -121,7 +121,19 @@ class PolicyDelta; end
       #     A single identity requesting access for a Cloud Platform resource.
       #     Follows the same format of Binding.members.
       #     Required
-      class BindingDelta; end
+      class BindingDelta
```

Comment:
I believe this change should also be applied to google-cloud-spanner/lib/google/cloud/spanner/admin/instance/v1/doc/google/iam/v1/policy.rb, yes?

Reply:
Done.